apps/plausiblestats/plausible_stats.star

```diff
@@ -0,0 +1,569 @@
+""")
+
+def main(config):
+    # For the "add" screen, show only the logo.
+    if config.get(DOMAIN_KEY) == None and config.get(PLAUSIBLE_API_KEY) == None:
+        return render_error_screen(None)
```

Comment:
It'd be sweet if the app could show an illustrative default graphic that shows off what the app looks like when fully set up. This would make it easier for users to understand what the app is all about, and to decide if they want to hop on over to Plausible.io, grab API keys and start exploring.

Reply:
Very true. I've revamped the demo screen to better reflect how the app works:

![screenshot-store](https://user-images.githubusercontent.com/199170/192011866-0af6d8d4-6d4a-4712-a225-875d36209eef.gif)